Microsoft/BotBuilder-Samples
# Add CoreBot to VSIX template

## Proposed Changes
  - Add `CoreBot` sample to VSIX templates
  - Update `Readme.md` file with info related to `CoreBot`
  - Also, fixes issue #1073
## Testing
1.  Open the `BotBuilderTemplatesVSIX.sln` in Visual Studio 2017
2.  Once the project is loaded, rebuild it. Once the re-build process is concluded, close the Visual Studio instance.
3.  Go to `{your project path}\bin\debug` folder and run the `BotBuilderVSIXV4.vsix` VSIX installer. Ensure that there isn't any Visual Studio instance open.
4. Once the installation process is concluded, open a Visual Studio 2017 instance and go to `File -> new -> project` and you will see the templates under the `Visual C# -> Cloud -> AzureBotService` category

![image](https://user-images.githubusercontent.com/43762887/51698129-36932f00-1fe8-11e9-8769-6108180c7c3d.png)

5. To test it in Visual Studio 2019, open a new instance and select  `Create New Project`. You will see the templates in the template selection window.

![image](https://user-images.githubusercontent.com/43762887/51698380-c89b3780-1fe8-11e9-8d6f-690ff56eb5b7.png)
